cogs/birthday/birthday.py

```diff
@@ -512,52 +543,100 @@ async def setSelfBirthday(self, ctx: Context, *, birthday: MonthDayConverter):
         birthdayStr = "{0:%B} {0:%d}".format(birthday)
 
         if birthdayConfig:
-            confirmationStr = (
-                f"Are you sure you want to set your birthday to "
-                f"{spoiler(bold(birthdayStr, escape_formatting=False), escape_formatting=False)}? "
-                "Only administrators and moderators can reset your birthday afterwards. "
-                f"Type {bold('`yes`', escape_formatting=False)} to confirm."
+            confirmationStr = "\n".join(
+                (
+                    f"Are you sure you want to set your birthday to "
+                    f"{spoiler(bold(birthdayStr, escape_formatting=False), escape_formatting=False)}? "
+                    "Only administrators and moderators can reset your birthday afterwards.",
+                    f"Type {bold('`yes`', escape_formatting=False)} to confirm.",
+                )
             )
 
-            await ctx.author.send(f"{headerWarn}: {confirmationStr}")
+            async def mainFlow(destination, carefree: bool = False):
+                # define the time for the message to live for
+                SENSITIVE_MSG_TTL = None if carefree else 10.0
+                try:
+                    await destination.send(
+                        f"{headerWarn}: {confirmationStr}",
+                        delete_after=SENSITIVE_MSG_TTL,
+                    )
+                except discord.Forbidden:
+                    # catch failed DM
+                    raise
+                # wait for answer
```

Comment:
nit: add a leading blank line for readability.
```suggestion

                # wait for answer
```

Reply:
Fixed